docs/design/hbar-limiter.md

```diff
@@ -178,6 +184,52 @@ classDiagram
     EthAddressHbarSpendingPlanRepository --> CacheService : uses
     IpAddressHbarSpendingPlanRepository --> CacheService : uses
 ```
+### Support flexible alerting mechanisms for spending thresholds
+The initial spending threshold of the Tier 3 General users will be a rough
```

Comment:
I might have missed it but how does alerting system for tier 2 and tier 1 look like? Why is only tier 3 being mentioned here?

Reply:
Good point.  It now has more detail.